docs/using-concourse/teams.any

```diff
@@ -266,11 +268,24 @@ as).
           --generic-oauth-client-secret $CLIENT_SECRET \\
           --generic-oauth-auth-url https://oauth.example.com/o/oauth2/auth \\
           --generic-oauth-token-url https://oauth.example.com/o/oauth2/token \\
-          --generic-oauth-auth-url-param=scope:read
+          --generic-oauth-auth-url-param scope:read
```

Comment:
Not redundant at all. Again, one is passed during the OAuth request (`generic-oauth-auth-url-param`) and the other is used for verification of the user in the response received (`generic-oath-scope`). Related, yet separate.